scipy/stats/_continuous_distns.py

```diff
@@ -6123,6 +6125,43 @@ def _stats(self, b, moments='mv'):
     def _entropy(self, c):
         return 1 + 1.0/c - np.log(c)
 
+    def fit(self, data, *args, **kwds):
+        data = np.asarray(data)
+
+        floc = kwds.pop('floc', None)
+
+        # borrowed from distn_infrastructure
+        # puts shape into f0
+        if self.shapes:
+            shapes = self.shapes.replace(',', ' ').split()
+            for j, s in enumerate(shapes):
+                key = 'f' + str(j)
+                names = [key, 'f' + s, 'fix_' + s]
+                val = _get_fixed_fit_value(kwds, names)
+                if val is not None:
+                    kwds[key] = val
+
+        shape = kwds.pop('f0', None)
+        fscale = kwds.pop('fscale', None)
+
+        _check_fit_input_parameters(data, args,
+                                    kwds, fixed_param=(shape, floc, fscale))
+
+        if floc is None or fscale is not None or np.any(data - floc < 0):
```

Comment:
@mdhaber 
`  if ... or np.any(data - floc < 0):`

Reply:
I was looking for the code `np.all(data - floc > 0)`.

I think you already thumbs upped my comment from @WarrenWeckesser that we can raise an error when the user's `floc` causes the data to go outside the valid range, right? This should do that, too, right?

Reply:
I used np.any since we talked about short circuit operators in my programming language class. I assumed that np.any would use that behavior and it might improve speed a bit over np.all.

Reply:
Yes, I'l was just explaining that you wrote `np.all(data - floc > 0)` above, so that's what I was looking for.